### PR TITLE
11474 Misc UI updates

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* /bcgov/entity#11474

*Description of changes:*
- updated column breakpoints (3/9 vs 2/10)
- added Select button to NAICS results
- updated hint text
- app version = 3.4.7

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/Registration/NaicsResult.vue [hunk 1]
```diff
@@ -1,11 +1,14 @@
 <template>
-  <v-row no-gutters class="naics-result" v-if="!!result" @click="emitClick(result)">
```
Comment (severinbeauvais) [inline src/components/Registration/NaicsResult.vue:2]: No need to emit `result`: parent already know which result this is (to select it).

File: src/components/Registration/NaicsResult.vue [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
+  <v-row no-gutters class="naics-result" v-if="!!result" @click="emitClick()">
     <v-col cols="2">
       <div class="result-code">{{result.code}}</div>
     </v-col>
 
     <v-col cols="10">
-      <label class="result-class-title">{{result.classTitle}}</label>
+      <div class="d-flex justify-space-between align-top">
+        <label class="result-class-title">{{result.classTitle}}</label>
+        <v-btn large depressed color="primary" class="mt-n2">Select</v-btn>
```
Comment (severinbeauvais) [inline src/components/Registration/NaicsResult.vue:10]: No need for `@click` handler: the event bubble up to the row, which handles it.

File: src/components/Registration/NatureOfBusiness.vue [hunk 3]
```diff
@@ -5,13 +5,13 @@
     :class="{ 'invalid-section': showErrors && !isFormValid }"
   >
     <v-row>
-      <v-col cols="12" sm="2">
+      <v-col cols="12" sm="3">
```
Comment (severinbeauvais) [inline src/components/Registration/NatureOfBusiness.vue:8]: I believe "small" is the correct breakpoint, but this makes the label row a little larger until it finally wraps. (Subtle UI tweak.)

**Review by cameron-freshworks — APPROVED** [review]
looks great!

**Pull request merged**